src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  margin-top: 20px;
+}
+
+.search-bar__big {
+  height: 70px;
+}
+
+.search-bar__small {
+  height: 42px;
+}
+
+.search-bar__logo--big {
+  width: 19px;
+  height: 19px;
+  padding-left: 26px;
+  padding-right: 17px;
+}
+
+.search-bar__logo--small {
+  width: 11px;
+  height: 11px;
+  padding-left: 13px;
+  padding-right: 9px;
+}
+
+.search-bar__input {
+  height: 19px;
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box` is duplicated, so make sense to move it to the `*{ box-sizing: border-box }`

Reply:
still not fixed
<img width="674" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/70220667/4ce27ed4-2d70-4de1-86d7-10e9c9f5784c">